### PR TITLE
[JAX FE]: Support jax.lax.lt, jax.lax.le and jax.lax.neg operation for JAX 

### PR DESCRIPTION
**Pull request by aku221b**
### Details:
 - support jax.lax.lt, jax.lax.le and jax.lax.neg operation

 - enable unit tests for new operations

### Tickets:
 - *None*

### CODE REVIEW

File: src/frontends/jax/src/op_table.cpp [hunk 1]
```diff
@@ -71,8 +72,11 @@ const std::map<std::string, CreatorFunction> get_supported_ops_jaxpr() {
             {"ge", op::translate_binary_op<v1::GreaterEqual>},
             {"gt", op::translate_binary_op<v1::Greater>},
             {"integer_pow", op::translate_integer_pow},
+            {"lt", op::translate_binary_op<v1::Less>},
+            {"le", op::translate_binary_op<v1::LessEqual>},
             {"max", op::translate_1to1_match_2_inputs<v1::Maximum>},
             {"mul", op::translate_1to1_match_2_inputs<v1::Multiply>},
+            {"neg", op::translate_negative},
```
Comment (rkazants) [inline src/frontends/jax/src/op_table.cpp:79]: please use `op::translate_1to1_match_1_input` as for other unary operations translated 1-to-1. So you don't need separate sources with translator

File: tests/layer_tests/jax_tests/test_negative.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    @pytest.mark.parametrize("input_shape", [
+        [10],
+        [2, 3],
+        [3, 4, 5],
+        [7, 16, 11, 20],
+        [6, 5, 4, 3, 2],
+        [1, 1, 9, 1]
+    ])
+    @pytest.mark.nightly
```
Comment (rkazants) [inline tests/layer_tests/jax_tests/test_negative.py:34]: please also check different input types, check `test_binary_comparison.py`

**Comment by aku221b** [comment]
@rkazants resolved comments.

File: tests/layer_tests/jax_tests/test_negative.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    @pytest.mark.parametrize("input_shape", [
+        [10],
+        [2, 3],
+        [3, 4, 5],
+        [7, 16, 11, 20],
+        [6, 5, 4, 3, 2],
+        [1, 1, 9, 1]
+    ])
+    @pytest.mark.parametrize('input_type', [np.int8,np.int16,np.int32, np.int64,np.float16, np.float32, np.float64])
```
Comment (rkazants) [inline tests/layer_tests/jax_tests/test_negative.py:41]:
```suggestion
    @pytest.mark.parametrize('input_type', [np.int8, np.int16, np.int32, np.int64,np.float16, np.float32, np.float64])
```

**Review by rkazants — APPROVED** [review]

**Comment by rkazants** [comment]
build_jenkins

**Comment by aku221b** [comment]
pushed commits with wrong author email, rebasing may alter commit history hence raising a fresh PR

**Pull request closed without merging**